substrate/frame/multi-asset-bounties/src/lib.rs

```diff
@@ -344,26 +346,29 @@ pub mod pallet {
 		///
 		/// The conversion from the native asset balance to the balance of an [`Self::AssetKind`] is
 		/// used in benchmarks to convert [`Self::BountyValueMinimum`] to the asset kind amount.
-		type BalanceConverter: ConversionFromAssetBalance<Self::Balance, Self::AssetKind, BalanceOf<Self, I>>
-			+ ConversionToAssetBalance<BalanceOf<Self, I>, Self::AssetKind, Self::Balance>;
+		type BalanceConverter: ConversionFromAssetBalance<Self::Balance, Self::AssetKind, Self::Balance>
+			+ ConversionToAssetBalance<Self::Balance, Self::AssetKind, Self::Balance>;
```

Comment:
I'm not sure it is necessary to force the native asset balance to be the same type as this pallet asset balances.

It is even slightly more error-prone, given the type will not warn when we forget to use `BalanceConverter`.
But I will not block on this, but still why is this changed in this PR, the pallet didn't work before?